tools/accuracy_checker/accuracy_checker/representation/segmentation_representation.py

```diff
@@ -201,6 +271,25 @@ def areas(self):
             areas.append(maskUtils.area(mask))
         return areas
 
+    def to_polygon(self):
+        if not self.raw_mask:
+            raise ValueError("Polygon can be found only for non-empty mask")
+
+        if not self.labels:
+            raise ValueError("Polygon can be found only for non-empty labels")
```

Comment:
I think in these case and above we can return empty dict (if you want, you can throw warning)